upp-elasticsearch-provisioner/README.md

```diff
@@ -63,48 +63,62 @@ docker run \
+Note that the following instructions assume that the source and target clusters have been created using `upp-elasticsearch-provisioner`, which will automatically create and register an S3 bucket for manual snapshots.
 
-    - Replace the `access_key` and `secret_key` values with the `upp-elasticsearch-provisioner` user keys for the appropriate **source** AWS account. These are listed in LastPass. Note that there are separate keys for Infra Prod, Content Test, and Content Prod - make sure you use the correct key.
+If your ES cluster has not been created by the provisioner, you will need to create and register an S3 bucket yourself - speak to your friendly neighbourhood Integration Engineer for help.
 
-    - Replace the `role` value with the the appropriate role ARN for the **source** AWS account:
+For the following steps, using [Postman](https://www.getpostman.com/) (or a similar tool) is strongly recommended, as it means you don't have to mess about with AWS credentials and certificates.
+
+You will need to pass the `content-containers-apps` credentials to Postman, which are available in LastPass.
+
+![](https://i.imgur.com/EYXPYCB.png)
+
+### Take a snapshot of the source ElasticSearch cluster
+
+- On your source cluster, send a `PUT` request to the following URL to create a snapshot, replacing your cluster hostname and snapshot name with appropriate values:
 ```
-ft-tech-infra-prod:
-arn:aws:iam::027104099916:role/upp-elasticsearch-backup-role
-ft-tech-content-platform-test:
-arn:aws:iam::070529446553:role/upp-elasticsearch-backup-role
-ft-tech-content-platform-prod:
-arn:aws:iam::469211898354:role/upp-elasticsearch-backup-role
+https://search-upp-concepts-source-cluster.eu-west-1.es.amazonaws.com/_snapshot/index-backups/20170526-efinlay
 ```
 
-- An example command is shown below:
+- Depending on the size of the source cluster, this may take some time to complete. You can check the progress of the snapshot by sending a `GET` request to the following URL:
 ```
-python register-es-snapshot-dir.py \
---region eu-west-1 \
---endpoint search-upp-concepts-source-cluster.eu-west-1.es.amazonaws.com \
---access_key example-access-key \
---secret_key example-secret-key \
---bucket upp-concepts-target-cluster-backup
---role arn:aws:iam::027104099916:role/upp-elasticsearch-backup-role
+https://search-upp-concepts-source-cluster.eu-west-1.es.amazonaws.com/_snapshot/index-backups/_all
 ```
 
-- For the following steps, using Postman is much easier than `curl`, as you need to pass the AWS credentials, and Postman allows you to do this with a nice GUI. :)
+### Copy snapshot data to target ElasticSearch cluster
 
-- Make sure that you pass the `content-containers-apps` credentials available in LastPass - not the `upp-elasticsearch-provisioner` credentials.
+Run the following commands on an AWS EC2 instance. You can run the commands locally, but it will be slower to sync the data to and from the buckets.
```

Comment:
`Run the following commands on an AWS EC2 instance` - EC2 instance closer to where the ES cluster is provisioned.